<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_novaVenda_cadastroCliente</class>
 <widget class="QDialog" name="fm_novaVenda_cadastroCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastrar cliente</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background:#ececec
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout" name="vlNome">
     <item>
      <widget class="QLabel" name="lbNome">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background:#e0e0e0;
color:#808080;
font-size:10pt;
border:none;
border-radius:6px;
}</string>
       </property>
       <property name="text">
        <string>Nome</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineNome">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit:enabled{background:#ffffff;}
QLineEdit:disabled{background:#e7e7e7;}

QLineEdit{
color:#303030;
font-size:9pt;
border-radius:7px;
border:1px solid black;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>150</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="vlTel">
     <item>
      <widget class="QLabel" name="lbTel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background:#e0e0e0;
color:#808080;
font-size:10pt;
border:none;
border-radius:6px;
}</string>
       </property>
       <property name="text">
        <string>Telefone (Apenas números)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineTel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit:enabled{background:#ffffff;}
QLineEdit:disabled{background:#e7e7e7;}

QLineEdit{
color:#303030;
font-size:9pt;
border-radius:7px;
border:1px solid black;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>11</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="vlCpf">
     <item>
      <widget class="QLabel" name="lbCpf">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background:#e0e0e0;
color:#808080;
font-size:10pt;
border:none;
border-radius:6px;
}</string>
       </property>
       <property name="text">
        <string>CPF (Apenas números)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineCpf">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit:enabled{background:#ffffff;}
QLineEdit:disabled{background:#e7e7e7;}

QLineEdit{
color:#303030;
font-size:9pt;
border-radius:7px;
border:1px solid black;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>11</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="vlCnpj">
     <item>
      <widget class="QLabel" name="lbCnpj">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background:#e0e0e0;
color:#808080;
font-size:10pt;
border:none;
border-radius:6px;
}</string>
       </property>
       <property name="text">
        <string>CNPJ (Apenas números)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineCnpj">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit:enabled{background:#ffffff;}
QLineEdit:disabled{background:#e7e7e7;}

QLineEdit{
color:#303030;
font-size:9pt;
border-radius:7px;
border:1px solid black;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>14</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <layout class="QVBoxLayout" name="vlCep">
     <item>
      <widget class="QLabel" name="lbCep">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background:#e0e0e0;
color:#808080;
font-size:10pt;
border:none;
border-radius:6px;
}</string>
       </property>
       <property name="text">
        <string>CEP (Apenas números)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineCep">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit:enabled{background:#ffffff;}
QLineEdit:disabled{background:#e7e7e7;}

QLineEdit{
color:#303030;
font-size:9pt;
border-radius:7px;
border:1px solid black;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QVBoxLayout" name="vlEndereco">
     <item>
      <widget class="QLabel" name="lbEndereco">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background:#e0e0e0;
color:#808080;
font-size:10pt;
border:none;
border-radius:6px;
}</string>
       </property>
       <property name="text">
        <string>Endereço</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEndereco">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit:enabled{background:#ffffff;}
QLineEdit:disabled{background:#e7e7e7;}

QLineEdit{
color:#303030;
font-size:9pt;
border-radius:7px;
border:1px solid black;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>150</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <layout class="QVBoxLayout" name="vlRef">
     <item>
      <widget class="QLabel" name="lbRef">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background:#e0e0e0;
color:#808080;
font-size:10pt;
border:none;
border-radius:6px;
}</string>
       </property>
       <property name="text">
        <string>Referência</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineRef">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit:enabled{background:#ffffff;}
QLineEdit:disabled{background:#e7e7e7;}

QLineEdit{
color:#303030;
font-size:9pt;
border-radius:7px;
border:1px solid black;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>150</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3">
    <spacer name="vs">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>478</width>
       <height>58</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btnGravar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>20</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:enabled{background:#7dbe50;}
QPushButton:disabled{background:#aaff7f;}
QPushButton:pressed{background:#aaff00;}
QPushButton{
color:#ffffff;
font-size:9pt;
border:none;
border-radius:10px;
}</string>
     </property>
     <property name="text">
      <string>Gravar novo</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineNome</tabstop>
  <tabstop>lineTel</tabstop>
  <tabstop>lineCnpj</tabstop>
  <tabstop>lineCep</tabstop>
  <tabstop>lineEndereco</tabstop>
  <tabstop>lineRef</tabstop>
  <tabstop>btnGravar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
